benchy

```diff
@@ -521,14 +521,16 @@ sysinfo() {
   disk_usage=$(echo "$df_output"| awk 'NR==2{ print $3 }'| convertsize)
   disk_percentage=$(echo "$df_output" | awk 'NR==2{ print $5 }')
   if is_openbsd; then
-      mem_info=$(grep -i "real mem" /var/run/dmesg.boot | cut -f2 -d"(" | cut -f1 -d")" | cut -f1 -d"MB")
-      mem_free=$(grep -i "avail mem" /var/run/dmesg.boot | cut -f2 -d"(" | cut -f1 -d")" | cut -f1 -d"MB")
-      mem_usage=$(echo "($mem_info-$mem_free)" | bc)
-      mem_percentage=$(echo "$mem_usage*100/$mem_free" | bc)
+      mem_info=$(($(sysctl -n hw.physmem)))
```

Comment:
Did you mean `mem_info=$(sysctl -n hw.physmem)` ?

Reply:
Nice catch! Forgot didn't do any arithmetic there.